ta/pkcs11/src/serializer.c

```diff
@@ -112,3 +112,29 @@ enum pkcs11_rc serialargs_get_session_from_handle(struct serialargs *args,
 
 	return PKCS11_CKR_OK;
 }
+
+/*
+ * serialize - serialize input data in buffer
+ *
+ * Serialize data in provided buffer.
+ */
```

Comment:
Should better remove the description since already done in **serializer.h** header file?

Reply:
OK